benchmarks/build.gradle.kts

```diff
@@ -59,6 +59,7 @@ android {
 
 baselineProfile {
     // This specifies the managed devices to use that you run the tests on.
+    managedDevices.clear()
```

Comment:
Why do you need to call clear() here?

Reply:
There are several devices added in a plugin and we only want to run BP generation only a single device